src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -529,43 +525,102 @@ public boolean verifyNewRankOptionsQuestionFormIsDisplayed() {
         WebElement contribForm = browser.driver.findElement(By.id("rankOptionsForm"));
         return contribForm.isDisplayed() && addNewQuestionButton.isDisplayed();
     }
+    
     public boolean verifyNewRankRecipientsQuestionFormIsDisplayed() {
         WebElement contribForm = browser.driver.findElement(By.id("rankRecipientsForm"));
         return contribForm.isDisplayed() && addNewQuestionButton.isDisplayed();
     }
-    
-    public boolean verifyEndDatesBeforeTodayAreDisabled() {
-        Calendar cal = Calendar.getInstance();
-        
-        // if today is the 1st day of the month, the datepicker cannot navigate to the previous month (yesterday) 
-        if (cal.get(Calendar.DATE) != 1) {
-            cal.add(Calendar.DATE, -1);       
 
-            // get month names
-            DateFormatSymbols dfs = new DateFormatSymbols();
-            String[] months = dfs.getMonths();        
+    public boolean isDatesOfPreviousCurrentAndNextMonthEnabled() throws ParseException {
+        return isDatesOfPreviousCurrentAndNextMonthEnabled(startDateBox) 
+               && isDatesOfPreviousCurrentAndNextMonthEnabled(endDateBox);
+    }
+
+    /**
+     * @param dateBox is a {@link WebElement} that triggers a datepicker
+     * @return true if the dates of previous, current and next month are
+     *         enabled, otherwise false
+     * @throws ParseException if the string in {@code dateBox} cannot be parsed
+     */
+    private boolean isDatesOfPreviousCurrentAndNextMonthEnabled(WebElement dateBox) throws ParseException {
+
+        Calendar previousMonth = Calendar.getInstance();
+        previousMonth.add(Calendar.MONTH, -1);
+
+        // Navigate to the previous month
+        if (!navigate(dateBox, previousMonth)) {
+            return false;
```

Comment:
No, there's no need to rename the method since it's still doing what its name says it does. Although, <code>areDates...</code> could be a better name.